backend/Origam.Server/Configuration/IdentityServerConfig.cs

```diff
@@ -56,6 +56,7 @@ public class IdentityServerConfig
         public bool CookieSlidingExpiration { get; } 
         public int CookieExpirationMinutes { get; }
         public string AuthenticationPostProcessor { get; }
+        public bool UseTokensForAuthenticationEverywhere { get; }
```

Comment:
I would change the wording. Better would be something like
`PrivateApiAuthentication = Cookie | Token`.